opal/group_test.go

```diff
@@ -168,72 +176,66 @@ func TestAccGroup_Reviewer(t *testing.T) {
+					resource.TestCheckResourceAttr(resourceName, "request_configuration.0.reviewer_stage.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "request_configuration.0.reviewer_stage.0.reviewer.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "request_configuration.0.reviewer_stage.0.operator", "AND"),
+					resource.TestCheckResourceAttr(resourceName, "request_configuration.0.reviewer_stage.0.require_manager_approval", "false"),
+					resource.TestCheckTypeSetElemNestedAttrs(resourceName, "request_configuration.0.reviewer_stage.0.reviewer.*", map[string]string{"id": knownCustomAppAdminOwnerID}),
 				),
 			},
+			// can't test this since we are deprecating the old way of passing in reviewers
+			// {
+			// 	Config: testAccGroupResource(baseName, baseName, ""),
+			// 	Check: resource.ComposeTestCheckFunc(
+			// 		resource.TestCheckResourceAttr(resourceName, "name", baseName),
+			// 		resource.TestCheckResourceAttr(resourceName, "request_configuration.0.reviewer_stage.#", "1"),
+			// 	),
+			// },
 			{
 				Config: testAccGroupResource(baseName, baseName, testReviewerStage("AND", false, knownOpalAppAdminOwnerID)),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(resourceName, "name", baseName),
-					resource.TestCheckResourceAttr(resourceName, "reviewer_stage.#", "1"),
-					resource.TestCheckResourceAttr(resourceName, "reviewer_stage.0.reviewer.#", "1"),
-					resource.TestCheckResourceAttr(resourceName, "reviewer_stage.0.operator", "AND"),
-					resource.TestCheckResourceAttr(resourceName, "reviewer_stage.0.require_manager_approval", "false"),
-					resource.TestCheckTypeSetElemNestedAttrs(resourceName, "reviewer_stage.0.reviewer.*", map[string]string{"id": knownOpalAppAdminOwnerID}),
-				),
-			},
-			{
-				Config: testAccGroupResource(baseName, baseName, ""),
-				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr(resourceName, "name", baseName),
-					resource.TestCheckResourceAttr(resourceName, "reviewer_stage.#", "0"),
+					resource.TestCheckResourceAttr(resourceName, "request_configuration.0.reviewer_stage.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "request_configuration.0.reviewer_stage.0.reviewer.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "request_configuration.0.reviewer_stage.0.operator", "AND"),
+					resource.TestCheckResourceAttr(resourceName, "request_configuration.0.reviewer_stage.0.require_manager_approval", "false"),
+					resource.TestCheckTypeSetElemNestedAttrs(resourceName, "request_configuration.0.reviewer_stage.0.reviewer.*", map[string]string{"id": knownOpalAppAdminOwnerID}),
 				),
 			},
+			// {
+			// 	Config: testAccGroupResource(baseName, baseName, ""),
+			// 	Check: resource.ComposeTestCheckFunc(
+			// 		resource.TestCheckResourceAttr(resourceName, "name", baseName),
+			// 		resource.TestCheckResourceAttr(resourceName, "request_configuration.0.reviewer_stage.#", "1"),
+			// 	),
+			// },
```

Comment:
this is the same as L213-220? if there isn't a plan to re-add it I would prefer to rm